contract/Cargo.toml

```diff
@@ -69,8 +73,16 @@ inherits = "release"
 [profile.zcash]
 inherits = "release"
 
+[profile.dogecoin]
+inherits = "release"
+
+[profile.dogecoin-testnet]
+inherits = "release"
+
 [features]
 default = ["bitcoin"]
+scrypt_hash = ["btc-types/scrypt_hash"]
```

Comment:
Do we really need a seperate scryp_hash flag?

Reply:
fixed: https://github.com/Near-One/btc-light-client-contract/pull/101/commits/c1efecbf7b83052d7a0512b9d6a386871420f11d